add_demo_src/demo.py

```diff
@@ -0,0 +1,26 @@
+class Demo:
+    def __init__(self, 
+                 name, 
+                 key, 
+                 url, 
+                 size=157.6*1024*1024, 
+                 heap=387*1024*1024, 
+                 compiler="", 
+                 engine="", 
+                 date="", 
+                 apis=["wasm", "URP", "WebGL 2", "linear-color-space"], 
+                 noVsync=True, 
+                 interactive=True, 
+                 mobile=True):
+        self.name = name
+        self.key = key
+        self.url = url
+        self.size = size
+        self.heap = heap
+        self.compiler = compiler
+        self.engine = engine
+        self.date = date
+        self.apis = apis
+        self.noVsync = noVsync
+        self.interactive = interactive
+        self.mobile = mobile
```

Comment:
I wonder if we need a Demo class at all? We could just use a regular dictionary object to represent a Demo?

Reply:
Either way works - the main thing I was after here is to avoid repetition that needs to list out all the properties in several places, which is a sign of redundant boilerplate in general.

Reply:
I respectfully disagree that it is redundant :) Happy to chat more about it if you want though